### ISSUE TITLE
新版本副本 以下 zone 失效的问题

### ISSUE THREAD

**Issue by zc149352394** [TeraProxy/Essentials#11]
if(mod.game.me.zone < 9000 && mod.settings.dungeonOnly) return

新副本将失效:
3012 守護者訓練所
3016 莉里斯城(單人)
3018 漆黑的泰內布利斯城堡(7人)
3019 扭曲的里安地下殿堂(7人)
3017 空洞的安塔洛斯深淵(7人)
3001 秘密基地(1人)
3002 凡貝爾宅邸(1人)
3003 邪教徒隱身處(1人)
3101 費爾奎娜巢穴
3201 燦爛的費爾奎娜巢穴

**Comment by zc149352394** [comment]
还有 第84行

**Comment by TeraProxy** [comment]
Those are zones that should also use the auto nostrum?

**Comment by zc149352394** [comment]
https://github.com/zc149352394/DataCenter_Final_TW.dat/blob/master/StrSheet_Dungeon-0.json


All Dungeons List


Maybe add this Zone

**Comment by TeraProxy** [comment]
```
    {
      "id": 8999,
      "string": "Velik's Sanctuary"
    },
    {
      "id": 3012,
      "string": "Training Ground"
    },
    {
      "id": 3016,
      "string": "Lilith's Trial"
    },
    {
      "id": 3018,
      "string": "Dark Reach Citadel (7-person)"
    },
    {
      "id": 3019,
      "string": "Grotto of Lost Souls (7-person)"
    },
    {
      "id": 3017,
      "string": "Antaroth's Abyss (7-person)"
    },
    {
      "id": 3001,
      "string": "Bastion of Lok (Solo)"
    },
    {
      "id": 3002,
      "string": "Sinestral Manor (Solo)"
    },
    {
      "id": 3003,
      "string": "Cultists' Refuge (Solo)"
    }
```

You are right. Those should be added. Having a manual list to update is nasty though so I should implement direct pulling of the dungeon IDs from the datacenter file.

**Comment by TeraProxy** [comment]
Proper dungeon status has been implemented in version 2.3.1
